README.md

```diff
@@ -414,25 +441,26 @@ terragrunt = {
 }
 ```
 
-The `include` block tells Terragrunt to use the exact settings from the `terraform.tfvars` file specified via the
-`path` parameter. It behaves exactly as if you had copy/pasted the contents of the root `terraform.tfvars` file into
-`mysql/terraform.tfvars`, but this approach is much easier to maintain! Note that if you include any other settings
-in the `terragrunt` block of a child `.tfvars` file, it will override the settings in the parent.
+The `include` block tells Terragrunt to use the exact same Terragrunt configuration from the `terraform.tfvars` file
+specified via the `path` parameter. It behaves exactly as if you had copy/pasted the Terraform configuration from 
+the root `terraform.tfvars` file into `mysql/terraform.tfvars`, but this approach is much easier to maintain! Note
+that if you include any other settings in the `terragrunt` block of a child `.tfvars` file, it will override the
+settings in the parent.
 
-The `terraform.tfvars` files above use two *helper functions*:
+The `terraform.tfvars` files above use two Terragrunt built-in functions:
 
-* `find_in_parent_folders()`: This helper returns the path to the first `terraform.tfvars` file it finds in the parent
+* `find_in_parent_folders()`: This function returns the path to the first `terraform.tfvars` file it finds in the parent
   folders above the current `terraform.tfvars` file. In the example above, the call to `find_in_parent_folders()` in
   `mysql/terraform.tfvars` will return `../terraform.tfvars`. This way, you don't have to hard code the `path`
   parameter in every module.
 
-* `path_relative_to_include()`: This helper function returns the relative path between the current `terraform.tfvars`
+* `path_relative_to_include()`: This function returns the relative path between the current `terraform.tfvars`
```

Comment:
I just thought that having the terminology of Terragrunt and Terraform be consistent would make sense and Terraform already called them "built-in functions" in their docs. Just semantics.